### PR TITLE
Update changelog for flux release 1.7.0

### PR DESCRIPTION
**Pull request by squaremo**

### CODE REVIEW

**Comment by dholbach** [comment]
Maybe list contributors for this release too? I could find `@squaremo, @dholbach, @stefanprodan, @justinbarrick, @sukrit007, @brandon-bethke-neudesic` in the commit logs since 1.6.0.

**Comment by squaremo** [comment]
> Maybe list contributors for this release too? I could find `@squaremo, @dholbach, @stefanprodan, @justinbarrick, @sukrit007, @brandon-bethke-neudesic` in the commit logs since 1.6.0.

Oh yes forgot about that. I also try to include people who have contributed in issues.

**Comment by dholbach** [comment]
Is there an easy way to figure out who contributed in issues since the last release other than an inbox? :-)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+This release has a soupçon of bug fixes. It gets a minor version bump,
+because it introduces a new flag, `--listen-metrics`.
+
+### Fixes
+
+- Updates to workloads using initContainers can now succeed
+  [weaveworks/flux#1351](https://github.com/weaveworks/flux/pull/1351)
+- Port forwarding to GCP (and possibly others) works as intended
+  [weaveworks/flux#1334](https://github.com/weaveworks/flux/issues/1334)
+- No longer falls over if the --git-path doesn't exist
```
Comment (hiddeco) [inline CHANGELOG.md]: Aesthetics: `--git-path` (with ``)

**Comment by squaremo** [comment]
> Is there an easy way to figure out who contributed in issues since the last release other than an inbox? :-)

No; it's pretty tedious. I also try to filter for commits that affected fluxd and fluxctl, rather than helm-op or the chart (which have their own releases). Maybe we should just have a thanks page, with all contributors to anything ever.

**Comment by dholbach** [comment]
All right. Don't want to block the release on the 'thank you' discussion. Feel free to go ahead with whatever you deem is best. I'll generally investigate querying Github for this info.

**Pull request merged**